xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/server/script/OfficeServerScriptService.java

```diff
@@ -147,7 +160,10 @@ public boolean stopServer()
     public String getServerState()
     {
         this.officeServer.refreshState();
-        return this.officeServer.getState().toString();
+        final String normalizedStatusKey =
+            CaseUtils.toCamelCase(this.officeServer.getState().name().toLowerCase(), false, '_');
```

Comment:
I don't know this api. Could you give an example of a state and what it translates it into? Might be good to put as a comment for explanation.

Reply:
Small note: we don't use `final`. It's an antipattern reported by SonarQube.

Reply:
Still we don't really use it FTM as a best practice so for code homogeneity I wouldn't use it. But if you care about it, you could send a proposal to use that to see what others think. What's important is that we all agree about our coding practices since there's no code ownership.